mqtt/mqtt-edgehub/src/auth/authorization/edgehub.rs

```diff
@@ -0,0 +1,307 @@
+use std::convert::Infallible;
+
+use mqtt_broker_core::{
+    auth::{Activity, AuthId, Authorization, Authorizer, Connect, Operation, Publish, Subscribe},
+    ClientId, ClientInfo,
+};
+
+#[derive(Debug, Default)]
+pub struct EdgeHubAuthorizer;
+
+#[allow(clippy::unused_self)]
+impl EdgeHubAuthorizer {
+    pub fn new() -> Self {
```

Comment:
just curious why parameterless `new` if we have `default`?

Reply:
during the active development phase, it is convenient to have new method. usually, it tends to accept some parameters over time. in this case, it may not require anything